### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by LyndonFan**
Addresses most issues in #95

- Fixed 2 instances of unnecessary else
- Fixed 2 instances of using range(len(...))
- Fixed 1 instance of code complexity by removing unnecessary "is not None" check
- Fixed 1 instance of code security by using requests.get instead of urllib.request.urlretrieve

There is still 1 instance of code complexity issue remaining. I have performed some minor refactoring, but it's still too complex according to codefactor.io.

### CODE REVIEW

**Comment by LyndonFan** [comment]
The new code has passed all the checks. Would it be possible to get a review?

**Review by Alexsandruss — APPROVED** [review]
I see no problems, let's wait for at least one additional approve.

File: daal4py_bench/requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ pandas < 1.3.0
 daal4py
 openpyxl
 tqdm
+requests
```
Comment (napetrov) [inline daal4py_bench/requirements.txt:6]: Might be missing some context here - why we need requests module? Per my understanding we do not suppose to use it?

Comment (LyndonFan) [reply daal4py_bench/requirements.txt:6]: This is to replace the urllib.request.urlretrieve used in load_data. The urllib library failed a check in code factor so I used requests instead.

**Review by napetrov — APPROVED** [review]

**Pull request merged**